src/api/spec/models/flag/specified_flags_spec.rb

```diff
@@ -16,8 +16,8 @@
     end
 
     it 'has default status on effective flag' do
-      expect(subject.effective_flag(nil, nil).status).to eq(Flag.default_status(flag_type))
-      expect(subject.effective_flag(repository_name, architecture_name).status).to eq(Flag.default_status(flag_type))
+      expect(subject.effective_flag(nil, nil).status).to eq(subject.default_flag(nil, nil).status)
```

Comment:
Sorry Lukas, I don't get it. `Flag.default_status(flag_type)` is just a one liner calling `FlagHelper.default_for(flag_name)` . The function you replace it with is a much more complex variant to call the same helper

Reply:
Puhh right, this Flag stuff confuses me, I changed it to use the helper directly, thanks for the hint

Reply:
https://www.youtube.com/watch?v=LeyofQK6tRw